packages/presentation/lib/screen/details/details_screen.dart

```diff
@@ -0,0 +1,363 @@
+                                  allowHalfRating: true,
+                                  itemCount: 5,
+                                  itemSize: Dimens.size30,
+                                  ratingWidget: RatingWidget(
+                                    full: SvgPicture.asset(ImagesPath.starFull),
+                                    half: SvgPicture.asset(ImagesPath.starHalf),
+                                    empty:
+                                        SvgPicture.asset(ImagesPath.starEmpty),
+                                  ),
+                                  itemPadding: const EdgeInsets.only(
+                                      right: Dimens.size2),
+                                  onRatingUpdate: (rating) {},
+                                ),
+                              ],
+                            ),
+                            const SizedBox(
+                              height: Dimens.size10,
+                            ),
+                            Container(
+                              height: Dimens.size40,
+                              margin: const EdgeInsets.only(
+                                top: Dimens.size10,
+                                right: Dimens.size6,
+                                left: Dimens.size6,
+                                bottom: Dimens.size24,
+                              ),
+                              padding: const EdgeInsets.all(Dimens.size4),
+                              decoration: BoxDecoration(
+                                border: Border.all(
+                                  color: PrimaryColors.primaryBorderColor,
+                                  width: Dimens.size2,
+                                ),
+                                borderRadius:
+                                    BorderRadius.circular(Dimens.size34),
+                              ),
+                              child: TabBar(
+                                unselectedLabelColor:
+                                    PrimaryColors.whiteWithOpacity80,
+                                labelColor: Colors.white,
+                                indicatorColor:
+                                    PrimaryColors.whiteWithOpacity80,
+                                indicator: BoxDecoration(
+                                  color: PrimaryColors.primaryColor,
+                                  borderRadius:
+                                      BorderRadius.circular(Dimens.size24),
+                                ),
+                                tabs: const [
+                                  Tab(
+                                    text: 'Details',
+                                  ),
+                                  Tab(
+                                    text: 'Reviews',
+                                  ),
+                                  Tab(
+                                    text: 'Showtime',
+                                  ),
```

Comment:
l10n

Reply:
for the details screen, do localization in scope of details task. for previous screens in scope of refactoring